pkg/controller/command/verifiable/command.go

```diff
@@ -332,3 +411,67 @@ func (o *Command) generatePresentation(rw io.Writer, vc *verifiable.Credential,
 
 	return nil
 }
+
+// SignPresentation sign the verifiable presentation
+func (o *Command) SignPresentation(didDoc *did.Doc, vc *verifiable.Credential) (*verifiable.Presentation, error) { // nolint:lll
+	var s signer
+
+	pk, err := getPublicKeyID(didDoc)
+	if err != nil {
+		return nil, err
+	}
+
+	s, err = newKMSSigner(o.kms, o.kResolver, pk)
+	if err != nil {
+		return nil, err
+	}
+
+	signingCtx := &verifiable.LinkedDataProofContext{
+		VerificationMethod:      pk,
+		SignatureRepresentation: verifiable.SignatureJWS,
+		SignatureType:           "Ed25519Signature2018",
+		Suite: ed25519signature2018.New(
+			suite.WithSigner(s)),
+	}
+
+	vp, err := vc.Presentation()
+	if err != nil {
+		return nil, fmt.Errorf("failed to create vp: %w", err)
+	}
+
+	err = vp.AddLinkedDataProof(signingCtx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to sign vc: %w", err)
+	}
+
+	return vp, nil
+}
+
+func getPublicKeyID(didDoc *did.Doc) (string, error) {
```

Comment:
Don't use this function

Reply:
as we discussed this is the default facility we are supporting. So we will keep it